### ISSUE TITLE
Suggestion: CDN for a raw JS lib import

### ISSUE THREAD

**Issue by paulocheque** [node-saml/xml-crypto#139]
What about convert the node JS files into one single JS file to be easily imported directly in the HTML?

**Comment by LoneRifle** [comment]
This is effectively blocked by #87, as we need to have it work without Node dependencies first. Marking as such.

Duplicate of #87
